wiki/Beatmap_Editor/Test_mode/id.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - editor
+  - beatmapping
+  - mapping
+---
+
+# Test mode
+
+![Screenshot tampilan test mode](img/test-mode-example.jpg "Menggunakan test mode dengan autoplay")
+
+**Test mode** adalah sebuah fitur dari [beatmap editor](/wiki/Beatmap_Editor) yang mengizinkan pembuat beatmap untuk memainkan [beatmap](/wiki/Beatmap) sendiri mereka dari beatmap editor untuk dicoba. Mode ini bisi diakses melalui tombol `Test` di bawah-kanan editor.
+
+![Tombol test mode](img/test-mode-button.png "Tombol test mode di beatmap editor")
+
+*Perhatian: Memainkan sebuah beatmap di test mode telah diketahui untuk menyebabkan lag atau framedrop untuk beberapa pengguna.*
+
+Memainkan sebuah beatmap di test mode berbeda dengan memainkannya secara normal, test mode tidak akan mengirim score, tidak menampilkan papan peringkat, tidak akan membiarkan pemain gagal, dan selalu akan menampilkan informasi berikut ini di bagian kiri pada layar:
+
+- [Timestamp](/wiki/Modding/Timestamp) beatmap sekarang.
+- [Muatan SB](/wiki/Beatmap_editor/SB_load).
+- Total jumlah 300, 100, 50, dan miss yang di dapatkan.
+- Persentase dari hit pada waktu yang salah.
```

Comment:
don't think the `hit error` here should be translated tho. sth like `persentase *hit error* yang diperoleh` would make more sense here

Reply:
the original text was "mistimed hit" and not "hit error" so i got a bit confused, but sure, we'll go with that instead since it doesn't sound as confusing o_O